team-manager/src/test/java/ua/knu/knudev/teammanager/service/AccountProfileServiceTest.java

```diff
@@ -96,15 +99,16 @@ void setUp() {
 
     @Test
     @DisplayName("Should register successfully when all inputs are valid")
+    @SneakyThrows
```

Comment:
You don't need it here

Reply:
Rewrote to try catch
I have getBytes method, it throws IOException